src/translations/pl.json

```diff
@@ -1,5 +1,28 @@
 {
   "FX Rate": "Konwerter Walut",
   "About": "O projekcie",
-  "Contact": "Kontakt"
+  "Contact": "Kontakt",
+  "ConvertCurrencyCalculatorHeader" : "Przelicz walutę",
+  "ConverterInfoFirstParagraph": "Wyliczenia wykonywane są na podstawie najnowszych danych udostępnionych przez <strong>Narodowy Bank Polski</strong>",
+  "ConverterInfoSecondParagraph": "Wartości przedstawione w serwisie stanowią medianę reprezentatywnych walut.",
+  "ConverterInfoDate": "Ostatnia aktualizacja: ",
+  "ConverterRateExchange": "Kurs wymiany: ",
+  "AboutFirstPhotoFirstParagraph": "<strong>CConverter</strong> to open-sourcowy projekt stworzony przez <strong>Codrill</strong>. Mając do dyspozycji 150 walut z całego świata przeliczenia stają się banalnie proste",
```

Comment:
Nice try! But there are some voices in the internet about avoiding using HTML markup directly in the translation files in React based projects:

https://stackoverflow.com/questions/39836369/html-tags-in-i18next-translation-files-in-react

Reply:
Btw. You have forgotten about HTML tags in en.json file 🌚🐸

Reply:
Done bro